### DISCUSSION TITLE
When to enable email notifications for ordinary commenters?

### DISCUSSION THREAD

**Discussion by Pathsis** [djyde/cusdis#158] [General]
When to enable email notifications for **ordinary** commenters? Since Cusdis is a replacement for Disqus, it seems that a modest email notification feature is necessary.

Cusdis is good enough, maybe it's ready for commercial use? I'm willing to start paying a reasonable fee to enable email notifications.

Continuing to disable email notifications will only make Cusdis lose users, which we don't want to do. Cusdis is great!

Please think about it.

**Comment by djyde** [comment]
My concern of opening email notification is spam. I'm not sure how to deal with it yet.

**Reply by Pathsis** [reply]
> My concern of opening email notification is spam. I'm not sure how to deal with it yet.

Happy to see your reply! Understand your concern and appreciate your concern. But email notifications are a necessary addition to a commenting system, not an unnecessary addition, just like we can't stop eating for fear of choking. You have asked how to deal with this problem, do you still have no idea?

Now that it's stalled, I worry that Cusdis will miss its opportunity to grow and deliver on its promise to replace Disqus. 

Visitors to my site have complained to me about not being able to receive notifications. For them, no notice is a very troublesome thing.

**Reply by Pathsis** [reply]
> My concern of opening email notification is spam. I'm not sure how to deal with it yet.

Currently, can self-hosting enable email notification for ordinary commenters?

**Comment by 12nick12** [comment]
Another great feature would be to be able to require email verification and some sort of captcha. hcaptcha is pretty handy and free.
